client/src/components/StarRatings.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const TealStar = styled.span`
+  color: #008489;
+  font-size: 18px;
+  display: flex;
+  padding-left: 75px; 
+`;
+
+const GrayStar = styled.span`
+  color: #d8d8d8;
+  font-size: 18px;
+  display: flex;
+  flex-direction: row;
+`;
+
+const ReviewName = styled.span`
+  color: #484848;
+  display: flex;
+  flex-direction: row;
+  font-family: Circular, -apple-system, system-ui, Roboto, "Helvetica Neue", sans-serif;
+`;
+
+const TwoRatings = styled.span`
+  display: flex;
+  flex-direction: row;
+  padding-left: 150px; 
+`
+
+const Line = styled.hr`
+  border-top: 1px solid Gainsboro;
+`;
+
+const StarRatings = ({rating}) => {
+  let accuracy = <TealStar>★★★★★</TealStar>;
+  let communication = <TealStar>★★★★★</TealStar>;
+  let cleanliness = <TealStar>★★★★★</TealStar>;
+  let location = <TealStar>★★★★★</TealStar>;
+  let checkin = <TealStar>★★★★★</TealStar>;
+  let value = <TealStar>★★★★★</TealStar>;
+  let ratings = [rating.accuracy, rating.communication, rating.cleanliness, rating.location, rating.checkin, rating.value];
+  let ratingName = [accuracy, communication, cleanliness, location, checkin, value];
+
+  for (let i = 0; i < ratings.length; i += 1) {
```

Comment:
Nice job getting it in a loop and not doing it all manually! You might even be able to store the different teal star/gray star nodes in an array so you don't have to do so many if statements